tmk_core/protocol/vusb/usbconfig.h

```diff
@@ -160,7 +166,9 @@ section at the end of this file).
 /* This macro (if defined) is executed when a USB SET_ADDRESS request was
  * received.
  */
+#ifndef USB_COUNT_SOF
 #define USB_COUNT_SOF                   1
+#endif
```

Comment:
```suggestion
#define USB_COUNT_SOF                   1
```

Reply:
Unless i have missed another config option, xd002 **has** to have this set to 0

Reply:
This define has to do with USB suspend stuff in USB 1.1. I guess it's probably not that useful for the XD002.